client-ui/src/app/components/header/header.component.html

```diff
@@ -23,6 +23,7 @@
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
         <div class="title">
             Data Lineage Tracking And Visualization
+            <span class="spline-version">|&nbsp;&nbsp;Version {{appVersion}}</span>
```

Comment:
Header title will become a very long string in that way. It is better to display it as a second string or in another place.

Reply:
Are you volunteering? :)

Reply:
Should I? 
According to me, it was good enough before. I do not see any reason to solve it at all so far. Isn't it better to leave it as it was before?

Reply:
No, it wasn't good enough
![image](https://user-images.githubusercontent.com/795479/84605380-08441700-ae9d-11ea-85b3-ce84bd836762.png)

Reply:
I cannot see that happening on full HD resolution. So you are solving here some edge case. And it is good enough if you are supporting desktop so far.

Reply:
If it bothering you, just add some semi-transparent background, and it will be good enough for now for these edge cases.

Reply:
Not everybody has full HD :)

Reply:
yes, you are right most of the people has higher resolution :)

Reply:
I have nothing to say more about that. I will mark that conversion as resolved.